lib/udf_findhotel_utils.rb

```diff
@@ -297,29 +297,31 @@ def get_bid_match_type(items):
           type:        :function,
           name:        :make_gha_click_batch_id,
           description: "Returns a unique identifier for a batch of clicks reported by Google Hotel Ads.",
-          params:      "click_type varchar(max), date_type varchar(max), google_site varchar(max), country varchar(max), device varchar(max)",
+          params:      "date_type varchar(max), google_site varchar(max), country varchar(max), device varchar(max), hotel_id varchar(max), checkin varchar(max), los bigint",
           return_type: "varchar(max)",
           body:        %~
             import hashlib
             import json
 
             key = {
-                "click_type": click_type,
                 "date_type": date_type,
                 "google_site": google_site,
                 "country": country,
-                "device": device}
+                "device": device,
+                "hotel_id": hotel_id,
+                "checkin": checkin,
+                "los": los}
```

Comment:
Shouldn't we convert `los` to string here ? Because when it will come from the url it will be a string and then the hash value will be different.
At least that's what we do for bing.

Reply:
Hmm, actually we don't. I think the only reason why we do it on bing, is to keep it compatible to the way it was done on polku-transform. Thinking about this, I realized that better to just have everything as strings, then we would never break a lucro job because a user sent a string instead of integer, we would only not match. So, I will change this function to accept only strings.